cfgov/v1/jinja2/v1/includes/organisms/tables/base.html

```diff
@@ -95,3 +95,5 @@
         {% endfor %}
     </tbody>
 </table>
+
+{{ value.caption }}
```

Comment:
Technically this should use Wagtail's [`| richtext`](https://docs.wagtail.org/en/stable/reference/jinja2.html#richtext) Jinja filter, [for example](https://github.com/cfpb/consumerfinance.gov/blob/874e173340ebd60f3bf4181dd53d09f8be29161f/cfgov/ask_cfpb/jinja2/ask-cfpb/_disclaimer.html#L7).

```suggestion
{{ value.caption | richtext }}
```

Excluding this probably doesn't have an effect for text formatting styles only (like bold). If there are any references to e.g. Wagtail pass then this is needed to properly expand those references.

Reply:
looks like `{{ value.caption | richtext }}` is causing at least one unit test failure; I really don't understand the relevant unit test code well enough to correct the error.